jobs/build/sync-for-ci/Jenkinsfile

```
@@ -127,15 +129,28 @@ node {
 
                 stage("push to s3") {
                     commonlib.syncRepoToS3Mirror("${LOCAL_SYNC_DIR}/", "${S3_SYNC_DIR}/")
+                    if ( commonlib.ocp4Versions.get(0) == ${REPOSYNC_DIR} ) {
```

I don't think you can put a bare `${}` like that. Have you tried this in hack space?
probably ought to be:
```suggestion
                    if ( commonlib.ocp4Versions.get(0) == REPOSYNC_DIR ) {
```